python/ffi/Cargo.toml

```diff
@@ -12,6 +12,6 @@ crate-type = ["cdylib"]
 failure = "0.1"
 libc = "0.2"
 ffi-utils = { git = "https://github.com/snipsco/snips-utils-rs", rev = "291ce1d" }
-snips-nlu-parsers-ffi-macros = { git = "https://github.com/purecloudlabs/snips-nlu-parsers-genesys.git", branch = "NLU-1383" }
+snips-nlu-parsers-ffi-macros = { git = "https://github.com/purecloudlabs/snips-nlu-parsers-genesys.git", tag = "snips-nlu-parsers-genesys-0.1.2" }
```

Comment:
any reason to not just tag with version number? `0.1.2`

Reply:
we already have a tag `0.1.2` from the original repo ( fork maintains them )

Reply:
ok, can we find a different / more succinct prefix then? `snips-nlu-parsers-genesys` just duplicates what we already know from the artefact. How about `0.1.2-genesys` or `genesys-0.1.2` ?

Reply:
👍 will go with `genesys-0.1.2`